pemFioi/conceptDisplay-1.0.js

```diff
@@ -1,3 +1,12 @@
+var channel = null;
+
+$(function() {
+  if (window.opener || window.parent) {
```

Comment:
Add `if (window.Channel)` as `Channel` will not be defined if jschannel is not imported.